Library/Homebrew/tab.rb

```diff
@@ -231,6 +241,18 @@ def spec
     source["spec"].to_sym
   end
 
+  def versions
+    source["versions"]
+  end
+
+  def stable_version
+    Version.create(versions["stable"]) if versions["stable"]
+  end
+
+  def devel_version
+    Version.create(versions["devel"]) if versions["devel"]
+  end
```

Comment:
Is there a reason there's both `Tab#stable_version` and `Tab#devel_version` but no`Tab#head_version`? This feels like a surprising omission to me.